admin/files/online-prezence/online-prezence-sort.js

```diff
@@ -0,0 +1,121 @@
+/* credits https://www.w3schools.com/howto/howto_js_sort_table.asp (slight changes) */
+function sortTable(col, table) {
```

Comment:
Tohle je teda masakr:D

Nebyla by nějaká udržovaná knihovna? Jedeme nad jQuery 3. Nebo možná něco má Bootstrap JS? Ten je tady tuším ve verzi 5.

Reply:
Knihovny teoreticky jsou, ale nedělá to přímo jQuery nebo Bootstrap, ale jsou to další nadstavby (min. desítky kB). Ten kód vypadá dlouhý kvůli těm komentářům, ale podle mě je docela v pohodě. Pokud bys i tak rád, můžu to udělat nějakou knihovnou, ale podle mě to fakt není potřeba.